libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -67,7 +67,7 @@ typedef void (^SFSnapshotViewControllerDismissalBlock)(UIViewController* snapsho
 
 /** Delegate protocol for handling foregrounding and backgrounding in Mobile SDK apps.
  */
-NS_SWIFT_NAME(SalesforceManagerDelegate)
+NS_SWIFT_NAME(SalesforceManagerDelegate) SFSDK_DEPRECATED(8.3, 9.0, "Will be removed.")
```

Comment:
The methods to add and remove delegates aren’t available on Swift so I deprecated the delegate too, let me know if there was a different intention